external/vcm/vcm/cubedsphere/coarsen_restarts.py

```diff
@@ -40,13 +43,27 @@
     SFC_DATA_Y_CENTER,
 )
 from .regridz import regrid_to_area_weighted_pressure, regrid_to_edge_weighted_pressure
+from .xgcm import create_fv3_grid
 
 FREEZING_TEMPERATURE = 273.16
 SHDMIN_THRESHOLD = 0.011
 STYPE_LAND_ICE = 16.0
 VTYPE_LAND_ICE = 15.0
 X_DIM = "xaxis_1"
 Y_DIM = "yaxis_1"
+SIGMA_BLEND = 0.9
+FRACTION_TRACERS = ["cld_amt"]  # Always are area-weighted.
+MIXING_RATIO_TRACERS = [
+    "sphum",
+    "liq_wat",
+    "rainwat",
+    "ice_wat",
+    "snowwat",
+    "graupel",
+    "o3mr",
+    "sgs_tke",
```

Comment:
I guess TKE is technically not a mixing ratio.  We could go with `NON_FRACTION_TRACERS` I suppose.